cfme/fixtures/pytest_selenium.py

```diff
@@ -305,6 +322,14 @@ def wait_for_ajax():
     """
 
     _thread_local.ajax_log_msg = ''
+    orig_events = get_events()
+
+    def _events_not_finished():
+        events = get_events()
+        not_finished = False
+        not_finished |= orig_events["maxTimeoutId"] >= events['minTimeoutId'] > 0
```

Comment:
That's correct. events['minTimeoutId'] is 0 only when there are no events or something went wrong and we were not able to read the value at all. Maybe I should have done it like this:
```
if events['minTimeoutId'] > 0:
  not_finished |= orig_events["maxTimeoutId"] >= events['minTimeoutId']
```
to make the purpose more obvious